azure/HandsOnTutorial/credentials.fsx

```diff
@@ -1,7 +1,7 @@
-﻿#I "../../packages/MBrace.Azure.Standalone/tools"
```

Comment:
This line is needed so that fsi loads the correct Azure dependencies. I added this because fsi would resolve older versions found in the Visual Studio installation folder.